bcipy/parameters/parameters.json

```diff
@@ -865,5 +865,45 @@
     "helpTip": "What should the name of the main log file be for each BciPy session",
     "recommended_values": "",
     "type": "str"
+  },
+  "Artifact Rejection": {
```

Comment:
lowercase parameter values

Reply:
gotcha. I've lowercased all parameter values that I added!